extensions/exposition/documentation/authorities.md

```diff
@@ -0,0 +1,53 @@
+# Authorities
+
+Authorities are a mechanism that allows serving multiple domains from a single instance of the
+application.
+
+## Definition
+
+The `authorities` definition is a map of authority identifiers to the `:authority` pseudo-header
+values.
+
+```yaml
+# context.toa.yaml
+
+exposition:
+  authorities:
+    one: the.one.com
+    two: the.two.com
+```
+
+## Ingress
+
+Each host in the authority definition is used to create a Kubernetes Ingress resource.
+
+> If the application is accessed with the `:authority` that does not match the authority definition,
```

Comment:
`:authority` is HTTP/2 pseudo-header so how can "application is accessed with with the `:authority`" happen on HTTP/1? What does this means?